src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -121,6 +121,10 @@ public class SnowflakeSinkConnectorConfig {
   public static final String SNOWPIPE_STREAMING_MAX_CLIENT_LAG =
       "snowflake.streaming.max.client.lag";
 
+  // This is the streaming max client lag which can be defined in config
```

Comment:
Nit, `streaming max channel size (in bytes)`

Reply:
The name seemed a bit long but I changed config to `snowflake.streaming.max.channel.size.in.bytes` to be more explicit.